the completions only install if bash is detected.
instead of adding bash to makedeps, just always install them
--
diff --git a/gio/meson.build b/gio/meson.build
index 462606f..bb3e19d 100644
--- a/gio/meson.build
+++ b/gio/meson.build
@@ -805,7 +805,7 @@ if host_system == 'windows'
   internal_deps += [ giowin32_lib ]
 endif
 
-if have_bash
+if true
   bash_comp_inst_dir = ''
   if bash_comp_dep.found()
     bash_comp_dir_override = bash_comp_dep.version().version_compare('>= 2.10') ? ['datadir', get_option('datadir')] : ['prefix', get_option('prefix')]
